### DISCUSSION TITLE
Routing profiles (+ Hiking & Mountain bike)

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#167] [Q&A] [answered]
Changes for simplification in routing settings:
- Each routing service has a separate screen with its options
The choices of one service do not affect other services!
- More routing profiles can be available (Hiking, Mountain bike)
- BRouter Hiking profile settings inside Cruiser

It should become more simple:
- We select a routing profile, e.g. Motorcycle
- We select a routing service, e.g. BRouter (offline)
- We only change the options of the selected router

GraphHopper options can be seen here:
- https://github.com/devemux86/cruiser/discussions/166#discussioncomment-5905241

What do you think?

<img src="https://github.com/devemux86/cruiser/assets/3484020/af5443d5-cf86-4826-ad57-266acbbd15d4" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/a4dd43a2-edd0-42ea-954b-f8f1489e204e" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/a9c02261-346e-4ddb-8fab-12e0fa24b345" width="33%"/>
<img src="https://github.com/devemux86/cruiser/assets/3484020/861feb34-dde6-46dc-8e47-5d3ec3388dc4" width="33%"/><img src="https://user-images.githubusercontent.com/3484020/236631825-7c7e3e56-d680-4146-ac7a-118fc0b9bc94.jpg" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/c4db0287-2e9a-4c7c-a729-907f7815560f" width="33%"/>

**Comment by DolomitesTom** [comment]
Sounds good

**Comment by Ciutnik** [comment]
Oh yes, this will make choosing the right setting for the user more clear. Walker, cyclist, motorcyclist - everyone will find something suitable for themselves, without worrying about the settings that are important only to others.
As a walker, cyclist and motorcyclist, I will be happy to test everything 🙂

**Comment by devemux86** [comment, marked as answer]
Detailed settings for each router are available in [Cruiser 3.0.20](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

Also new application profiles for Hiking and Mountain bike.
